### ISSUE TITLE
New release?🤔

### ISSUE THREAD

**Issue by kloczek** [swig/swig#2802]
https://github.com/swig/swig/compare/v4.2.0...master shows +70 commits since last release with substantial number of some fixes.
Do you have any plans to flush all those changes and release 4.2.1? 🤔

**Comment by wsfulton** [comment]
Yup, I was thinking of releasing the current state in a week or so.

**Comment by kloczek** [comment]
Thx 👍
